pkg/driver/node/credentialprovider/provider_pod.go

```diff
@@ -19,8 +19,10 @@ import (
 )
 
 const (
-	serviceAccountTokenAudienceSTS = "sts.amazonaws.com"
-	serviceAccountRoleAnnotation   = "eks.amazonaws.com/role-arn"
+	serviceAccountTokenAudienceSTS         = "sts.amazonaws.com"
+	serviceAccountTokenAudiencePodIdentity = "pods.eks.amazonaws.com"
+	serviceAccountRoleAnnotation           = "eks.amazonaws.com/role-arn"
+	podIdentityCredURI                     = "http://169.254.170.23/v1/credentials"
```

Comment:
As a comment, could you please provide doc link where it says that URI should be this value?

Reply:
Done, thanks! The handling of custom addresses will be implemented in a follow-up PR